packages/@aws-cdk/custom-resources/test/aws-custom-resource/aws-custom-resource.test.ts

```diff
@@ -342,6 +342,98 @@ test('getData', () => {
   });
 });
 
+test('fails when getData is used with `ignoreErrorCodesMatching`', () => {
+
+  const stack = new cdk.Stack();
+  expect(() => {
+
+    const resource = new AwsCustomResource(stack, 'AwsSdk', {
```

Comment:
To ensure that it's `resource.getData()` that throws I would move the `resource` declaration out of the `expect` here.

Reply:
Good call.